conf/messages.en

```diff
@@ -2124,6 +2124,16 @@ checkNetProfitLoss.title.profit = Net business profit: £{0}
 checkNetProfitLoss.p1.individual = We’re now going to ask you about any final adjustments to your your net business profit. This will give you your your total taxable profit or loss.
 checkNetProfitLoss.p1.agent = We’re now going to ask you about any final adjustments to your client’s your net business profit. This will give you their your total taxable profit or loss.
 
+previousUnusedLosses.title.individual = Did {0} have any unused losses in previous years?
+previousUnusedLosses.subHeading.individual = These are unused losses that you have told us about in previous tax returns and chose to carry forward.
+previousUnusedLosses.subHeading.agent = These are losses that your client told us about in previous tax returns and chose to carry forward.
+previousUnusedLosses.error.required.individual = Select yes if your business made any unused losses in previous years
+previousUnusedLosses.error.required.agent = Select yes if your client’s business made any unused losses in previous years
+
+unusedLossAmount.title.individual = Enter the total amount of unused loss
```

Comment:
Are these errors still being used by the form or can they be deleted?

Reply:
Still waiting to hear about them.